app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,69 @@
+//= require_tree .
+//= require_self
+
+@import url("http://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,400,300,600");
+
+body {
+  margin: 60px;
+  font-family: 'Open Sans', sans-serif;
+  p {
+    font-size: 1em;
+    font-weight: 300;
+  }
+  h1 {
+    font-size: 2.4em;
+    font-weight: 600;
+  }
+  h2 {
+    font-size: 2em;
+    font-weight: 600;
+  }
+  h3 {
+    font-size: 1.6em;
+    font-weight: 600;
+  }
+  h4 {
+    font-size: 1.2em;
+    font-weight: 600;
+  }
+  a {
+    color: #0066FF;
+    font-size: 1em;
+    font-weight: 300;
+    text-decoration: none;
+    border-bottom: 1px solid #0066FF;
+  }
+}
+
+.description {
+  font-style: italic;
+}
+
+.auth-links {
+  display: block;
+  position: fixed;
+  right: 100px;
+  top: 60px;
+}
+
+.home-link {
+  display: block;
+  position: fixed;
+  right: 180px;
+  top: 60px;
+}
+
+.alert-boxes {
+  position: absolute;
+  top: 0;
+  left: 50%;
+  transform: translateX(-50%);
+  padding: 10px;
+  border-radius: 2px;
+  .alert-alert {
+    color: red;
+  }
+  .alert-notice {
+    background: yellow;
```

Comment:
Color `yellow` should be written in hexadecimal form as `#ffff00`<br>Color literals like `yellow` should only be used in variable declarations; they should be referred to via variable everywhere else.